src/app/creation/creation.component.html

```diff
@@ -0,0 +1,115 @@
+<form>
+<mat-tab-group mat-align-tabs="start">
+    <mat-tab label="Race">
+        <mat-form-field appearance="fill">
+            <mat-label>Character Race</mat-label>
+            <mat-select [(ngModel)]="selectedRace" name="race" (selectionChange)="displayRaceDescription($event.value)">
+                <mat-option *ngFor="let race of raceData" [value]="race.name">
+                {{race.name}}
+                </mat-option>
+            </mat-select>
+        </mat-form-field>
+        <div class="body">{{raceDescription}}</div>
+
+    </mat-tab>
+    <mat-tab label="Class">
+        <mat-form-field appearance="fill">
+            <mat-label>Character Class</mat-label>
+            <mat-select [(ngModel)]="selectedClass" name="class" (selectionChange)="displayClassDescription($event.value)">
+                <mat-option *ngFor="let class of classData" [value]="class.name">
+                {{class.name}}
+                </mat-option>
+            </mat-select>
+        </mat-form-field>
+        <div class="body">{{classDescription}}</div>
+    </mat-tab>
+    <mat-tab label="Attributes">
+        <div class=body>
+            <p>Normally your ability scores are generated randomly, but for new players we recommend you use the Standard array. 
+                Use the following scores and assign them to your attributes: 15, 14, 13, 12, 10, 8.
```

Comment:
nit:  might be beneficial to generate the end of the comment based on the list of attribute scores in the component, that way we don't have to keep them in sync if they  ever change